torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -64,13 +64,29 @@ def _torch_ones(sizes, requires_grad=False):
 # rref tensor equals to the given grad.
 def _compare_owner_value(context_id, rref, grad):
     grads = dist_autograd.get_gradients(context_id)
-    return torch.equal(grads[rref.local_value()], grad)
+    x = grads[rref.local_value()]
+    if x.is_sparse:
+        assert grad.is_sparse
+        x = x.to_dense()
+        grad = grad.to_dense()
```

Comment:
Is this because `torch.equal` does not support sparse tensors and there is no `torch.sparse.equal`? If so, shall we create an issue to track?